### DISCUSSION TITLE
Blurry elements and "low FPS" issues

### DISCUSSION THREAD

**Discussion by Myslatron** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3609] [Q&A] [answered]
Disclaimer: I'm fairly new with WPF and MDXT so please be patient with me.

The issue what I'm having is first that my application looks blurry but not all the time and not everything.
Some examples:
My splashscreen is perfectly sharp:
![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/167078389/20108653-b7ad-455c-bccf-2bf9cd05f49f)
In my MainWindow there are some elemnt blurry some sharp
![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/167078389/8633c83d-e3bf-4c28-9cac-fde5b7fc291c)

I made my application DPI aware but it did not fix everything, what I'm missing what is wrong with my app?
Also the Demo App runs smoothly 60 FPS and my application sometimes is slow and stuttering, is there anything I can do with that?

Thank you for any suggestions.

**Comment by MichelMichels** [comment, marked as answer]
This could be related to #3612  and #3601 , also see PR #3616

**Reply by Myslatron** [reply]
Thank you, this fixed the bluryness!

**Comment by MichelMichels** [comment]
@Myslatron have you checked if the recent PR fixed your problem?
